docusaurus/docs/React/release-guides/emoji-picker.mdx

```diff
@@ -0,0 +1,78 @@
+---
+id: new-emoji-picker-integration
+sidebar_position: 3
+title: EmojiPicker Integration (11.0.0)
```

Comment:
Maybe also the title could be changed to something more v11-ish? Will we be including those changes like removal of deprecated components?